packages/ui/vite.config.ts

```diff
@@ -43,7 +43,10 @@ export default defineConfig({
   build: {
     emptyOutDir: true,
     lib: {
-      entry: getAllUIComponents(),
+      entry: {
+        'src/tailwindConfig': join(__dirname, 'src', 'tailwindConfig.ts'),
```

Comment:
thanks for noticing! Instead of expanding on the `getAllUIComponents` function, I moved the `tailwindConfig` to its own directory and it got immediately taken by the current build process

Reply:
for now it's a unique entry, don't see a reason to prepare for the future possible cases